docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -162,10 +159,12 @@ dune --deploy ./build/hello hello
 
 ### Interacting with your contract
 
-To interact with your contract you will send a transaction on your local EOS node. Transactions on EOS are made up of 
-`actions`, so we will send a single action to your contract.
+Transactions on EOS consist of 
+`actions`. To interact with your contract send a single action to your contract.
+
+You also need to create a test account from which to send the action.
 
-We will also create a test account to send the action from.
+Use the following commands to create a test account and send an action:
```

Comment:
```
Transactions on EOS consist of actions. To interact with your contract send a single action to your contract.

You also need to create a test account from which to send the action.

Use the following commands to create a test account and send an action:
```

This is still very verbose, and hard to read. `consist` is also a 12th-grade reading level, we are aiming lower than that to help non-English speakers.

Reply:
The first line of the example is a definition, typically written "A transaction on EOS is ..."  I could write this sentence as "A transaction on EOS is ... and contains actions.

The red highlighted sentence is passive due to the helping verb and ends in a preposition.

Reply:
Done